infrastructure/tooling/src/importer/util.js

```diff
@@ -137,20 +96,17 @@ exports.readAzureTableInChunks = async function* ({azureCreds, tableName, utils,
       const results = await table.queryEntities(tableName, tableParams);
```

Comment:
Would this throw if we are not able to connect to azure? Should we handle this case?

Reply:
Yeah, I think there are a number of ways this script could fail and adding handling for all of those feels like it's not a good use of time -- it's just adding code that can't really be tested and will probably just fail in a different way if something does go wrong.  Or it will mask errors and we'll wait for hours for the import to complete when it's sitting there hitting the same error over and over.  We've had good luck so far, aside from moments when we deliberately killed the pg server, so I think we're OK where we are..